diagnostic.rb

```diff
@@ -1,16 +1,37 @@
 require 'csv'
 # Diagnostic answers to:
+  rails g model License surname:string given_name:string gender:string height:integer weight:integer born_on:date eye_color:string hair_color:string
```

Comment:
This will generate the correct model and migration files. How would you run the migration?